src/Harald.IntegrationTests/Facades/Slack/SlackFacadeTest.cs

```diff
@@ -5,18 +5,20 @@
 using System.Threading.Tasks;
 using Harald.WebApi.Infrastructure.Facades.Slack;
 using Harald.WebApi.Infrastructure.Serialization;
+using Xunit.Priority;
 
 namespace Harald.IntegrationTests.Facades.Slack
 {
+    [TestCaseOrderer(PriorityOrderer.Name, PriorityOrderer.Assembly)]
     public class SlackFacadeTest
     {
-        [Fact]
+        [Fact, Priority(0)]
         public async Task CreateChannel_Given_valid_input_Should_create_channel()
         {
             // Arrange
             var httpClient = GetHttpClient();
             var sut = new SlackFacade(httpClient, new JsonSerializer(), new SlackHelper());
-            var channelName = "janie-test";
+            var channelName = "ded-team-one";
 
             // Act
             var createChannelResponse = await sut.CreateChannel(channelName);

```

Comment:
It will fail on line 27 due to the API returning an error. A separate PR(https://github.com/dfds/harald/pull/11) takes care of this issue where a channel name already exists. Even with that PR merged in, this test will still fail, since the generated name will be different than expected(line 28). 

Not sure what to do about this yet, maybe create a separate test that runs after this one, checking that the duplicate name handling is correct?